css/index.css

```diff
@@ -65,12 +65,35 @@ h1, h2, h3, h4, h5 {
 p {
```

Comment:
Nice work splitting the file into a CSS file for each page, makes your code more organized and readable.